src/spikeinterface/core/sortinganalyzer.py

```diff
@@ -531,7 +534,8 @@ def create_zarr(cls, folder, sorting, recording, sparsity, return_scaled, rec_at
             zarr_root.create_dataset("recording", data=zarr_rec, object_codec=numcodecs.Pickle())
         else:
             warnings.warn(
-                "SortingAnalyzer with zarr : the Recording is not json serializable, the recording link will be lost for future load"
+                "SortingAnalyzer with zarr : the Recording is not json serializable, "
+                "the recording link will be lost for future load"
```

Comment:
This is a bit meta, but this warning isn't super helpful. Heberto has converted me to the camp that a warning should help. If I want to do zarr and I want the recording link how do I "fix" this warning. If i'm a new user I likely won't know what the pros vs cons of losing my recording link even mean. Maybe not in this PR, but I'll try to do another assert/warning PR for terminology soon.